usaspending_api/api_contracts/contracts/v2/idvs/amounts/award_id.md

```diff
@@ -24,6 +24,7 @@ Get aggregated award counts and funding amounts for IDV contracts
         * `child_award_base_exercised_options_val`: 297285.59 (required, number)
         * `child_total_account_outlay`: 0 (required, number)
         * `child_total_account_obligation`: 0 (required, number)
+        * `child_award_total_outlay`: 0 (required, number)
```

Comment:
Contract should say `nullable` or return should default to 0.

Reply:
Agreed, since `null` will indicate that the award doesn't meet the date logic outlined in the ticket